frontend/components/FieldAsset/FieldAsset.css

```diff
@@ -0,0 +1,28 @@
+.container {
+  width: 100%;
+  margin-top: 9px;
+  height: 40px;
+  line-height: 25px;
+  font-size: 15px;
+  box-sizing: border-box;
+  border: 1px solid #e4e4e4;
+  border-radius: 5px;
+  outline: 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.container[preview] {
```

Comment:
We should only style by class name, not attribute. If you want to add a styling block based on whether a condition is met, you should use the `Style` class. See [this](https://github.com/dadi/publish/blob/master/frontend/components/Button/Button.jsx#L99) as an example.